src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -210,6 +210,7 @@ const fetchAchievementFromBasket = async () => {
 };
 
 watch(addedLoan, async () => {
+	if (!addedLoan?.value?.id) return;
 	if (myKivaExperimentEnabled.value && !isGuest.value) {
 		await fetchBasketData();
 		fetchPostCheckoutAchievements([...loansIdsInBasket.value, addedLoan.value?.id]);
```

Comment:
Here we are still adding duplicate loan IDs. Because there is an `await` on `fetchBasketData`, I think the line here needs to be just `fetchPostCheckoutAchievements(loansIdsInBasket.value);`.